jwt-tutorial-003/src/main/java/org/jwt/client/ApplicationEntryPoint.java

```diff
@@ -42,7 +42,7 @@ public void onModuleLoad() {
 
 	private void initServerConnection() {
 		try {
-			new ServerConnectionBuilder().setWebSocketEnabled(false).build();
+			new ServerConnectionBuilder().setServerSentEventsEnabled(false).setWebSocketEnabled(false).build();
```

Comment:
Ähm. Wir wollen doch WS als Default haben. Das geht also genau in die falsche Richtung.

Reply:
Huchje, erstmal gar nicht gesehen dass du auch auf dem PR bist, schöne neue Welt. 😛 
Das Tutorial war auch vorher schon an dieser Stelle davon ausgegangen dass Longpoll verwendet wird, d.h. man müsste auch an anderen Stellen auf Websocket umstellen, die von vorne rein nur Longpoll konnten. 

Florian und ich haben nachhher noch ne Session hierzu, da werde ich den Vorschlag einbringen.

Reply:
Hierzu https://levigo.de/support/browse/JWT-2973 erstellt, in diesem Pull Request wird aber nur das Tutorial auf 5.8.0.0 wieder lauffähig gemacht mit alter Funktionalität (longpoll).